postgres-appliance/callback_aws.py

```diff
@@ -67,11 +61,15 @@ def main():
         associate_address(ec2, sys.argv[1], instance_id)
 
     tags = {'Role': role}
-    tag_instance(ec2, instance_id, tags)
+    tag_resource(ec2, instance_id, tags)
+
+    tags.update({'Instance': instance_id})
+    name_tag = 'spilo_' + cluster
 
-    tags.update({'Instance': instance_id, 'Name': 'spilo_' + cluster})
     volumes = list_volumes(ec2, instance_id)
-    tag_volumes(ec2, volumes, tags)
+    for v in volumes:
+        tags_to_update = tags if 'Name' in v.tags else dict(tags, Name=name_tag)
```

Comment:
Well, we didn't do that before.  If we start now, that will introduce inconsistencies.